config/sablier/sablier.yml

```diff
@@ -0,0 +1,20 @@
+provider:
+  name: docker
+server:
+  port: 10000
+  base-path: /
+storage:
+  file:
+sessions:
+  default-duration: 5m
```

Comment:
I believe that this config should be configurable, because you could work on the code and don't use env much more time. You will lose console history if it will be closed by stoping environment

Reply:
Agreed, implemented the same way as `traefik.yml` for consistency. It can be configured via `${WARDEN_HOME_DIR}/etc/sablier/sablier.yml`, Although it will be overwritten on `svc up` calls. 

Opened PR #813 which looks to migrate the Traefik config to environment variables. Where users can override / expand the configuration via the `~/.warden/docker-compose.yml` file. Sablier can be refactored to the same approach if approved.